frontend/test/metabase/scenarios/auditing/subscriptions.cy.spec.js

```diff
@@ -1,12 +1,19 @@
-import { restore } from "__support__/e2e/helpers/e2e-setup-helpers";
-import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
 import {
+  restore,
   modal,
   popover,
-} from "__support__/e2e/helpers/e2e-ui-elements-helpers";
-import { describeEE } from "__support__/e2e/helpers";
+  describeEE,
+  getFullName,
+} from "__support__/e2e/helpers";
+
+import { USERS } from "__support__/e2e/cypress_data";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
 
 const { ORDERS_ID } = SAMPLE_DATABASE;
+const { admin, nodata } = USERS;
+
+const adminFullName = getFullName(admin);
+const ADMIN_ID = 1;
```

Comment:
Don't we have the ID via `admin.id` or something like that?

Reply:
Nope.
I could get user ID via request, but that's an overkill for now, and also out of the current scope.

However, your point stands and we will take care of every single hard coded ID in the future.